src/site/stages/build/process-cms-exports/generator-cms-content-model/generators/app/index.js

```diff
@@ -32,7 +32,11 @@ const allSnakeCasedPropertyNames = obj =>
   new Set(
     _.flatten(
       Object.keys(obj).map(key => {
-        if (typeof obj[key] === 'object' && !Array.isArray(obj[key]))
+        if (
+          obj[key] &&
```

Comment:
:confused: What happened that necessitated this? Was there a null value? (Because `typeof null === 'object'`. Thanks JavaScript.)

Reply:
Yeah that's exactly what was happening.  `typeof null === 'object'` turns out to be truthy so things would just break.